google/cloud/forseti/scanner/scanners/iap_scanner.py

```diff
@@ -111,12 +111,27 @@ def instance_group_network_port(backend_service, instance_group):
         Returns:
             NetworkPort: how the service communicates with backends
         """
-        port = int(backend_service.port)
-        if backend_service.port_name:
-            for named_port in instance_group.named_ports or []:
-                if named_port.get('name') == backend_service.port_name:
-                    port = int(named_port.get('port'))
-                    break
+
+        # Field 'port' from backend service has been deprecated in favor of
+        # portName. PortName is required when the load balancing scheme is
+        # EXTERNAL. When the load balancing scheme is INTERNAL, this field
+        # is not used, it has the same behavior of port so we can just use
+        # portName to get the port from instance group.
+
+        if not backend_service.port_name:
+            # If backend_service.port_name is None, it's INTERNAL.
```

Comment:
nit: can you move this comment to above the `if` statement?

```
# If backend_service.port_name is None, it's INTERNAL.
if not backend_service.port_name:
    return None
```

Reply:
Updated.